.changeset/perfect-trainers-tie.md

```diff
@@ -0,0 +1,7 @@
+---
+'@spreadshirt/backstage-plugin-s3-viewer': patch
+'@spreadshirt/backstage-plugin-s3-viewer-backend': patch
+'@spreadshirt/backstage-plugin-s3-viewer-common': patch
+---
+
+Security update
```

Comment:
Please highlight the most important packages updated here and that they were pointed out by Dependabot.

Reply:
I also allowed Dependabot to open pull requests for such alerts (in the GitHub settings for this repo), let's see if that helps.  We can then merge those as they happen and release them with each Backstage update.

Reply:
Cool! I did forgot to enable that when I enabled the dependabot in the past, so I didn't see those alerts until now.

I have updated the commit changeset with a list of the critical and high alerts fixed